Resources/Prototypes/_StarLight/Entities/Objects/Specific/Medical/handheld_brigmedic_crew_monitor.yml

```diff
@@ -0,0 +1,31 @@
+- type: entity
+  name: BrigBuddy™ 5000
+  parent: [ BaseHandheldComputer, BaseRestrictedContraband ]
+  id: HandheldBrigmedicCrewMonitor
+  description: So advanced it only tracks security personnel! Does not monitor emotional stability or competence levels of security members. Use at your own risk!
+  components:
+  - type: Sprite
+    sprite: _Starlight/Objects/Specific/Medical/handheldbrigmediccrewmonitor.rsi
```

Comment:
handheldbrigmediccrewmonitor 

https://github.com/user-attachments/assets/10eb2c9d-2763-472c-83cd-cd1bddd9a9a5

Reply:
fixed to 
![image](https://github.com/user-attachments/assets/fe02d04b-ece3-491b-b745-e4e84a053133)